src/Parser/YaccParser.mly

```diff
@@ -270,20 +460,20 @@ data_rec_rest
 /* ========================================================================= */
 
 def
-: KW_IMPLICIT TLID implicit_ty_args type_annot_opt
+:  KW_IMPLICIT TLID implicit_ty_args type_annot_opt
```

Comment:
Another unnecessary change. My advice is to run `git diff` before each commit, to find such accidental changes.